lib/tfgen/testdata/TestGenerate_AccessMonitoringRule.golden

```diff
@@ -0,0 +1,37 @@
+resource "teleport_access_monitoring_rule" "monitoring-rule" {
```

Comment:
Hey @bernardjkim 👋🏻 I've included this example as I believe it's the resource you're going to be working on soon?